GOVERNANCE.md

```diff
@@ -0,0 +1,179 @@
+
+### Voting Procedure
+
+Elections will be held using a ranked choice voting system which supports voter privacy.  Specific voter tooling will be selected by the Election Officers before the election starts, with this selection to be approved by the Steering Committee. The top vote-getters will be elected to the open seats, with the exceptions for company representation discussed below.
+
+### Limitations on Company Representation
+
+No more than 2 seats may be held by employees of the same organization (or  
+conglomerate, in the case of companies owning each other). If an election results in greater than 2 employees of the same organization being selected, the lowest vote getters from any particular employer will be removed until representation on the committee is down to 2.
+
+If employers change because of job changes, acquisitions, or other events, in a way that would yield too many seats being held by employees of the same organization, and the next regular election is more than 10 weeks away, sufficient members of the committee must resign until only 2 employees of the same employer are left. If it is impossible to find sufficient members to resign, all employees of that organization will be removed and new special elections held. 
+
+In the event of a question of company membership (for example evaluating independence of corporate subsidiaries) a majority of all non-involved Steering Committee members will decide.
+
+## Vacancies
+
+In the event of a resignation or other loss of an elected Steering Committee member, a new vote for that seat is started by Election Officers.
+
+A special election for that position will be held as soon as possible, unless the regular Steering Committee election is less than 10 weeks away. Eligible voters from the most recent election will vote in the special election. Eligibility will not be redetermined at the time of the special election. Any replacement Steering Committee member will serve out the remainder of the term of the person they are replacing, regardless of the length of that remainder.
+
+## Changes to the Governance
+
+Changes to this Governance may be proposed via a Pull Request on the Governance itself.  
+Amendments are accepted with 2/3rd majority consent of the committee as per the [Decision Process](#decision-process) outlined above.
+
+Proposals and amendments to the charter are available for at least a period of one week for comments and questions before a vote will occur.
+
+## Authority, Facilitation, and Decision Making
+
+Ideally, most decisions will be made at the lowest possible level within the project: within individual working groups. When this is not possible, the Steering Committee can help facilitate a conversation to work through the contended issue. When facilitation by the Steering Committee does not resolve the contention, the Steering Committee may have to decide.
```

Comment:
I'd put the SC later. Let things bubble up to e.g. dev summit without calling on SC for everything

Reply:
@gouthamve and I conclude that deciding "within individual working groups" includes dev summits and we don't have to explicitly state where decisions have to be made until SC is brought in.